### PR TITLE
Feature/1854 cookie consent

### PR DESCRIPTION
**Pull request by richard-jones**
adds the cookie consent banner to the site

### CODE REVIEW

File: portality/templates/doaj/cookie_consent.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div id="cookie-consent-banner">
+    DOAJ requires cookies to provide all of its features.
+    By using our website, you agree to our use of cookies. <a href="#">View our Privacy Notice</a> |
```
Comment (Steven-Eardley) [inline portality/templates/doaj/cookie_consent.html]:
```suggestion
    By using our website, you agree to our use of cookies. <a href="{{ url_for('doaj.privacy') }}">View our Privacy Notice</a> |
```

**Comment by Steven-Eardley** [comment]
Also, I suggest this can be a hotfix - so long as the behaviour is correct (which it is), a screenshot may suffice for review

**Comment by richard-jones** [comment]
Required changes are made, so when you get a moment @Steven-Eardley if you could release this that would be great.

**Comment by Steven-Eardley** [comment]
Released 2019-03-18.

**Pull request closed without merging**